stdlib/public/core/Array.swift

```diff
@@ -1788,7 +1788,7 @@ extension Array {
   /// and enums.
   ///
   /// The following example copies bytes from the `byteValues` array into
-  /// `numbers`, an array of `Int`:
+  /// `numbers`, an array of `Int32`; on a [little-endian](https://developer.mozilla.org/en-US/docs/Glossary/Endianness) machine, the result is as shown:
```

Comment:
Please wrap to 80 characters:

```suggestion
  /// `numbers`, an array of `Int32`; on a [little-endian][1] machine, the
  /// result is as shown:
  ///
  ///     [Example code]
  ///
  /// [1]: https://developer.mozilla.org/en-US/docs/Glossary/Endianness
```